doc/mayastor-client.md

```diff
@@ -101,7 +101,7 @@ fine writing to it as it were a local disk.
 ```bash
 > fallocate -l 2GiB /data/file.img
 > mayastor-client nexus create `uuidgen -r` 1GiB 'aio:///data/file.img?blk_size=512 aio:///dev/sdb'
```

Comment:
Should change this to reflect that children are now to be passed as separate arguments?

Reply:
Has this been changed? - The two children are still passed as a single argument, and this will no longer work.